src/components/Navbar/SettingsDrawer/AudioSection.tsx

```diff
@@ -37,6 +39,7 @@ const AudioSection = () => {
   const dispatch = useDispatch();
   const selectedReciter = useSelector(selectReciter, shallowEqual);
   const enableAutoScrolling = useSelector(selectEnableAutoScrolling);
+  const [playbackRate, setPlayBackRateState] = useState(DEFAULT_PLAYBACK_RATE);
 
   // given the reciterId, get the full reciter object.
```

Comment:
Nice! this all looks good. The only change is that we want to save and persist the playback speed in redux. 

Users who customize the playback speed would usually like to maintain that speed cross session.